### ISSUE TITLE
I want a example code to use the embedding projector in tensorboard by tensorflow-2.0.

### ISSUE THREAD

**Issue by jiawei6636** [tensorflow/tensorflow#32879]
I want a example code to use the embedding projector in tensorboard by tensorflow-2.0.

**Comment by ravikyram** [comment]
@jiawei6636 
Can you please go through below link and see if it helps you. Thanks!
https://www.tensorflow.org/tutorials/text/word_embeddings

**Comment by jiawei6636** [comment]
> @jiawei6636
> Can you please go through below link and see if it helps you. Thanks!
> https://www.tensorflow.org/tutorials/text/word_embeddings

I want to use the local tensorboard, not the online embedding projector. I can use the local tensorboard embedding projector in TF1.0, but I don't how to use it in TF2.0

**Comment by rmothukuru** [comment]
@jiawei6636,
Can you please refer [this link](https://stackoverflow.com/a/41370610/11530462) for visualizing Embedding Projector Locally and let us know if it helps. Thanks!

**Comment by jiawei6636** [comment]
> @jiawei6636,
> Can you please refer [this link](https://stackoverflow.com/a/41370610/11530462) for visualizing Embedding Projector Locally and let us know if it helps. Thanks!

No, it is TF1.0 code.

**Comment by rmothukuru** [comment]
@jiawei6636,
Can you please check [this link](https://github.com/tensorflow/tensorboard/issues/2471)  and let us know if it is helpful. Thanks!

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and the `awaiting response` label was assigned. Is this still an issue?

**Comment by jiawei6636** [comment]
> It has been 14 days with no activity and the `awaiting response` label was assigned. Is this still an issue?

Yes, this is still an issue. But I can solve it by [Pytorch(There](https://pytorch.org/docs/stable/tensorboard.html) is a convenient API in Pytorch). I think you should give me a  similar solution.

**Comment by jiawei6636** [comment]
> It has been 14 days with no activity and the `awaiting response` label was assigned. Is this still an issue?

Tensorboard is a good tool in Tensorflow's ecosystem, but the Pytorch uses it better than Tensorflow. 
 : )

**Comment by jiawei6636** [comment]
> > It has been 14 days with no activity and the `awaiting response` label was assigned. Is this still an issue?
> 
> Your answers are all perfunctory. Tensorboard is a good tool in Tensorflow's ecosystem, but the Pytorch uses it better than Tensorflow.
> : )

**Comment by mehta-a** [comment]
> @jiawei6636
> Can you please go through below link and see if it helps you. Thanks!
> https://www.tensorflow.org/tutorials/text/word_embeddings

I do not want to use http://projector.tensorflow.org/ but visualize my vectors/embeddings in board running locally. Can you share code for tf2.0. The issue is `from tensorflow.contrib.tensorboard.plugins import projector` command is invalid. :(

**Comment by asears** [comment]
The following works for me, though TF 1 vs 2 example updates would be much appreciated!

```
import tensorflow as tf
from tensorboard.plugins import projector

help(projector)
```

The help and examples need to be updated for 2.0.

**Comment by paloha** [comment]
In another issue I have posted some examples on how to make Tensorboard Projector work in both TF2.0 and TF2.1 in both Non-Eager and Eager modes. Please check [https://github.com/tensorflow/tensorboard/issues/2471#issuecomment-580423961](url). Hope this helps.

**Comment by manivaradarajan** [comment]
We have updated the Projector documentation with a TF 2.0 example: https://www.tensorflow.org/tensorboard/tensorboard_projector_plugin
